src/components/dynamic-image/DynamicImage.tsx

```diff
@@ -52,7 +54,7 @@ const fadeIn = (
     if (image === placeholder) {
       ref.current.style.backgroundColor = 'unset'
       // Apply an empty url here so we can use the background image prop to set a gradient
```

Comment:
mind dropping this comment since it no longer applies?

Reply:
Good catch, done!